src/components/form/range/dual_range.test.tsx

```diff
@@ -176,56 +176,5 @@ describe('EuiDualRange', () => {
 
       expect(component).toMatchSnapshot();
     });
-
-    test('should be set only for the min input', () => {
```

Comment:
This and the tests below were still too specific for aria-label, and really just ensuring that  the input props get passed down, doesn't error out, is all that's needed here so I simplified the version above and removed these and below.